tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | superuser |
+            | password         | super1234 |
+        And the user has logged in with username "superuser" and password "super1234"
+
+
+    Scenario: user creates a space
+        When the user creates a space "user's Space"
```

Comment:
What is this "user" that's creating space? If there's a concept of `admin` here then maybe changing it  like this would make more sense?
```suggestion
        When the admin creates a space "user's Space"
```